app/controllers/v0/institutions_controller.rb

```diff
@@ -24,7 +24,7 @@ def autocomplete
     def index
       @meta = {
         version: @version,
-        count: search_results.count,
+        count: search_results.load.size,
```

Comment:
This causes a SQL SELECT that loads returns every search result.  
The `COUNT` select that this replaces is much faster.
These search results are not used anywhere else (if you revert the `count_field` changes below; see my comment there), so the previous `.count` is much preferred.